code/__DEFINES/actions.dm

```diff
@@ -1,8 +1,50 @@
-// Checks to see if the mob is able to use their hands, or if they are blocked by cuffs or stuns
+
+///Action button checks if hands are unusable
 #define AB_CHECK_HANDS_BLOCKED (1<<0)
-// Checks to see if the mob is incapacitated by stuns or paralysis effects
-#define AB_CHECK_INCAPACITATED (1<<1)
-// Checks to see if the mob is standing
+///Action button checks if user is immobile
+#define AB_CHECK_IMMOBILE (1<<1)
+///Action button checks if user is resting
 #define AB_CHECK_LYING (1<<2)
-// Checks to see if the mob in concious
+///Action button checks if user is conscious
 #define AB_CHECK_CONSCIOUS (1<<3)
+///Action button checks if user is incapacitated
+#define AB_CHECK_INCAPACITATED (1<<4)
+///Action button checks if user is jaunting
+#define AB_CHECK_PHASED (1<<5)
+
+//Bitfield is in /_DEFINES/_globablvars/bitfields.dm for reasons
+
+
+///Action button triggered with right click
+#define TRIGGER_SECONDARY_ACTION (1<<0)
```

Comment:
Does this currently work with our context menu? I assume this is for trigger flags when right clicking action buttons, but does it do anything?

If not, I recommend commenting it out for now

Reply:
I forgot to remove this define when removing all mentions of right clicking